resources/historic.php

```diff
@@ -0,0 +1,926 @@
+    'alphabeticCode' => 'GEK',
+    'currency' => 'Georgian Coupon',
+    'minorUnit' => 0,
+    'numericCode' => 268,
+  ),
+  'DDM' => 
+  array (
+    'alphabeticCode' => 'DDM',
+    'currency' => 'Mark der DDR',
+    'minorUnit' => 0,
+    'numericCode' => 278,
+  ),
+  'DEM' => 
+  array (
+    'alphabeticCode' => 'DEM',
+    'currency' => 'Deutsche Mark',
+    'minorUnit' => 0,
+    'numericCode' => 276,
+  ),
+  'GHC' => 
+  array (
+    'alphabeticCode' => 'GHC',
+    'currency' => 'Cedi',
+    'minorUnit' => 0,
+    'numericCode' => 288,
+  ),
+  'GHP' => 
+  array (
+    'alphabeticCode' => 'GHP',
+    'currency' => 'Ghana Cedi',
+    'minorUnit' => 0,
+    'numericCode' => 939,
+  ),
+  'GRD' => 
+  array (
+    'alphabeticCode' => 'GRD',
+    'currency' => 'Drachma',
+    'minorUnit' => 0,
+    'numericCode' => 300,
+  ),
+  'GNE' => 
+  array (
+    'alphabeticCode' => 'GNE',
+    'currency' => 'Syli',
+    'minorUnit' => 0,
+    'numericCode' => 324,
+  ),
+  'GNS' => 
+  array (
+    'alphabeticCode' => 'GNS',
+    'currency' => 'Syli',
+    'minorUnit' => 0,
+    'numericCode' => 324,
+  ),
+  'GWE' => 
+  array (
+    'alphabeticCode' => 'GWE',
+    'currency' => 'Guinea Escudo',
+    'minorUnit' => 0,
+    'numericCode' => 624,
+  ),
+  'GWP' => 
+  array (
+    'alphabeticCode' => 'GWP',
+    'currency' => 'Guinea-Bissau Peso',
+    'minorUnit' => 0,
+    'numericCode' => 624,
+  ),
+  'ITL' => 
+  array (
+    'alphabeticCode' => 'ITL',
+    'currency' => 'Italian Lira',
+    'minorUnit' => 0,
+    'numericCode' => 380,
+  ),
+  'ISJ' => 
+  array (
+    'alphabeticCode' => 'ISJ',
+    'currency' => 'Old Krona',
+    'minorUnit' => 0,
+    'numericCode' => 352,
+  ),
+  'IEP' => 
+  array (
+    'alphabeticCode' => 'IEP',
+    'currency' => 'Irish Pound',
+    'minorUnit' => 0,
+    'numericCode' => 372,
+  ),
+  'ILP' => 
+  array (
+    'alphabeticCode' => 'ILP',
+    'currency' => 'Pound',
+    'minorUnit' => 0,
+    'numericCode' => 376,
+  ),
+  'ILR' => 
+  array (
+    'alphabeticCode' => 'ILR',
+    'currency' => 'Old Shekel',
+    'minorUnit' => 0,
+    'numericCode' => 376,
+  ),
+  'LAJ' => 
+  array (
+    'alphabeticCode' => 'LAJ',
+    'currency' => 'Pathet Lao Kip',
+    'minorUnit' => 0,
+    'numericCode' => 418,
+  ),
+  'LVL' => 
+  array (
+    'alphabeticCode' => 'LVL',
+    'currency' => 'Latvian Lats',
+    'minorUnit' => 0,
+    'numericCode' => 428,
+  ),
+  'LVR' => 
+  array (
+    'alphabeticCode' => 'LVR',
+    'currency' => 'Latvian Ruble',
+    'minorUnit' => 0,
+    'numericCode' => 428,
+  ),
+  'LSM' => 
+  array (
+    'alphabeticCode' => 'LSM',
+    'currency' => 'Loti',
+    'minorUnit' => 0,
+    'numericCode' => 426,
+  ),
+  'ZAL' => 
+  array (
+    'alphabeticCode' => 'ZAL',
+    'currency' => 'Financial Rand',
+    'minorUnit' => 0,
+    'numericCode' => 991,
+  ),
+  'LTL' => 
+  array (
+    'alphabeticCode' => 'LTL',
+    'currency' => 'Lithuanian Litas',
+    'minorUnit' => 0,
```

Comment:
Minor unit is not 0. It's 2. Seems that all historical currencies are set to 0.

Reply:
Can you do another PR?

Reply:
Unfortunately the information is not in the xmls that https://www.currency-iso.org provide. Is there any other trustful source of information that we can use?

Reply:
Wow, I haven't noticed it at all!

Reply:
What we can do here, is to fill this in by hand from Wiki page in some sort of XML and then parse it to all those formats. Those changes are quite rare, so might not be a bad idea I think to do this by hand.

Reply:
I think there also is a Symfony package that also busy with building a repository.